sdk/generic-sdk-core/generic-core/src/main/java/com/generic/core/implementation/http/rest/SwaggerMethodParser.java

```diff
@@ -568,20 +568,16 @@ private static String serialize(ObjectSerializer serializer, Object value) {
         if (value instanceof String) {
             return (String) value;
         } else if (value.getClass().isPrimitive()
+            || value.getClass().isEnum()
```

Comment:
Enum and ExpandableEnum both could still have the case where they are wrapping `null` and `toString` returns `null` instead of `"null"`.

Reply:
Wasn't that added to Azure Core because of Jackson? Is that something we want to do all the time?